### PR TITLE
tests: clean legacy

### PR DESCRIPTION
**Pull request by guits**
clean some legacies since quay.ceph.io migration

Signed-off-by: Guillaume Abrioux <gabrioux@redhat.com>

### CODE REVIEW

**Comment by guits** [comment]
jenkins test centos-non_container-lvm_batch

**Comment by guits** [comment]
jenkins test centos-non_container-collocation

**Comment by guits** [comment]
jenkins test centos-non_container-all_in_one

**Comment by guits** [comment]
jenkins test centos-non_container-all_in_one

**Comment by guits** [comment]
jenkins test centos-container-all_in_one

**Comment by guits** [comment]
jenkins test centos-container-collocation

**Comment by guits** [comment]
jenkins test centos-container-external_clients

**Comment by guits** [comment]
jenkins test centos-container-ooo_collocation

**Comment by guits** [comment]
jenkins test centos-container-purge

File: tests/functional/all_daemons/group_vars/all [hunk 1]
```diff
@@ -35,7 +35,9 @@ handler_health_osd_check_delay: 10
 mds_max_mds: 2
 dashboard_admin_password: $sX!cD$rYU6qR^B!
 grafana_admin_password: +xFRe+RES@7vg24n
+ceph_docker_registry: quay.ceph.io
+ceph_docker_image: ceph-ci/daemon
+ceph_docker_image_tag: latest-octopus
```
Comment (dsavineau) [inline tests/functional/all_daemons/group_vars/all:40]: latest-master

**Review by dsavineau — CHANGES_REQUESTED** [review]

**Comment by guits** [comment]
jenkins test centos-non_container-all_daemons

**Comment by guits** [comment]
jenkins test centos-container-collocation

**Comment by guits** [comment]
jenkins test centos-non_container-all_in_one

**Comment by guits** [comment]
jenkins test centos-non_container-collocation

**Comment by guits** [comment]
jenkins test centos-non_container-collocation

**Comment by guits** [comment]
jenkins test centos-non_container-switch_to_containers

**Comment by guits** [comment]
jenkins test centos-container-update

**Comment by guits** [comment]
jenkins test centos-non_container-purge

**Comment by guits** [comment]
jenkins test centos-non_container-collocation

**Comment by guits** [comment]
jenkins test centos-container-update

**Comment by guits** [comment]
jenkins test centos-non_container-collocation

**Comment by guits** [comment]
jenkins test centos-non_container-switch_to_containers

**Comment by guits** [comment]
jenkins test centos-container-update

**Comment by guits** [comment]
jenkins test centos-non_container-collocation

**Comment by guits** [comment]
jenkins test centos-non_container-switch_to_containers

**Comment by guits** [comment]
unrelated failuers

**Pull request merged**